cypress/e2e/participant/default-tests/playCVA.cy.js

```diff
@@ -4,13 +4,26 @@ import { isCurrentVersion } from '../../../support/utils';
 const app = '@bdelab/roar-multichoice';
 
 describe('ROAR - Written Vocabulary Playthrough', () => {
-  it('Plays CVA', () => {
-    cy.wrap(isCurrentVersion(app)).then((isCurrentVersion) => {
+  it('Plays CVA with username/password combo', () => {
+    cy.wrap(!isCurrentVersion(app)).then((isCurrentVersion) => {
       if (isCurrentVersion) {
         cy.log(`Did not detect a new version of ${app}, skipping test.`);
       } else {
         cy.log(`Detected a new version of ${app}, running test.`);
-        playWrittenVocabulary();
+        playWrittenVocabulary({ auth: 'username' });
+      }
+    });
+  });
+});
+
+describe('ROAR - Written Vocabulary Playthrough', () => {
+  it('Plays CVA with Clever auth', () => {
+    cy.wrap(isCurrentVersion(app)).then((isCurrentVersion) => {
+      if (!isCurrentVersion) {
+        cy.log(`Did not detect a new version of ${app}, skipping test.`);
+      } else {
+        cy.log(`Detected a new version of ${app}, running test.`);
+        playWrittenVocabulary({ auth: 'clever' });
       }
     });
   });
```

Comment:
```suggestion
describe('ROAR - Written Vocabulary Playthrough', () => {
  it('Plays CVA with username/password combo', () => {
    cy.wrap(!isCurrentVersion(app)).then((isCurrentVersion) => {
      if (isCurrentVersion) {
        cy.log(`Did not detect a new version of ${app}, skipping test.`);
      } else {
        cy.log(`Detected a new version of ${app}, running test.`);
        playWrittenVocabulary({ auth: 'username' });
      }
    });
  });
  it('Plays CVA with Clever auth', () => {
    cy.wrap(isCurrentVersion(app)).then((isCurrentVersion) => {
      if (!isCurrentVersion) {
        cy.log(`Did not detect a new version of ${app}, skipping test.`);
      } else {
        cy.log(`Detected a new version of ${app}, running test.`);
        playWrittenVocabulary({ auth: 'clever' });
      }
    });
  });
});

```
We can move all of the it-blocks into the single describe-block and they will run sequentially.

I'll just leave this one comment but we can do this for all of the tests when testing both forms of auth.

Reply:
Good point, I've made the refactor.